app/assets/javascripts/components/coordinates-search-form.js

```diff
@@ -0,0 +1,65 @@
+(function(window, $) {
+  'use strict';
+
+  var disabledButtonClassName = 'usa-button-disabled',
+      errorMessageHTML = '<div class="usa-alert usa-alert-error" role="alert"><div class="usa-alert-body"><p class="usa-alert-text">There was a problem determining your location. Please reload the page and try again.</p></div></div>';
```

Comment:
Possibly, yes. This advice will account for timeouts (which I've experienced) as well as users choosing _not_ to share their location (or otherwise dismissing the browser's permission request). Reloading the page will get the browser to re-display the permission request should the user choose to search by location again.